src/test/java/io/supertokens/test/multitenant/api/TestApp.java

```diff
@@ -507,4 +508,80 @@ public void testDefaultRecipesEnabledWhileCreatingApp() throws Exception {
         assertTrue(tenant.get("thirdParty").getAsJsonObject().get("enabled").getAsBoolean());
         assertTrue(tenant.get("passwordless").getAsJsonObject().get("enabled").getAsBoolean());
     }
+
+    @Test
+    public void testInvalidTypedValueInCoreConfigWhileCreatingApp() throws Exception {
+        if (StorageLayer.getStorage(process.getProcess()).getType() != STORAGE_TYPE.SQL) {
+            return;
+        }
+
+        if (StorageLayer.isInMemDb(process.getProcess())) {
+            return;
+        }
+
+        String[] properties = new String[]{
+                "access_token_validity", // long
+                "access_token_validity", // long
+                "access_token_validity", // long
+                "access_token_validity", // long
+                "disable_telemetry", // boolean
+                "postgresql_connection_pool_size", // int
+                "mysql_connection_pool_size", // int
+        };
+        Object[] values = new Object[]{
+                "abcd", // access_token_validity
+                "",
+                "null",
+                null,
+                "abcd", // disable_telemetry
+                "abcd", // postgresql_connection_pool_size
+                "abcd", // mysql_connection_pool_size
+        };
+
```

Comment:
confirm that the same message type is shown when refresh token is < access token.